environment-minimum.yml

```diff
@@ -0,0 +1,15 @@
+name: mpd-mini
+channels:
+  - conda-forge
+dependencies:
+  - python=3.10
+  - geopandas-base
+  - fiona
```

Comment:
ah I see fiona has been added in here

Reply:
Yes, fiona and rtree had to be added explicitly